maison-de-l-orient-et-de-la-mediterannee.csl

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" demote-non-dropping-particle="sort-only" default-locale="fr-FR" page-range-format="expanded">
+<info>
+    <title>Maison de l'Orient et de la Méditerranée (French)</title>
+    <id>http://www.zotero.org/styles/maison-de-l-orient-et-de-la-mediterannee</id>
+    <link href="http://www.zotero.org/styles/maison-de-l-orient-et-de-la-mediterannee" rel="self"/>
```

Comment:
@Lugnot, I spot a small error in the style's file name, style ID, and "self" link. "mediterannee" should be "mediterranee", right? (double "r", single "n")

Can you fix it in all three place? (see https://help.github.com/articles/renaming-a-file/ for help on renaming the file)